core/src/main/scala/kafka/admin/BrokerApiVersionsCommand.scala

```diff
@@ -231,6 +232,7 @@ object BrokerApiVersionsCommand {
           CommonClientConfigs.SECURITY_PROTOCOL_CONFIG,
           ConfigDef.Type.STRING,
           CommonClientConfigs.DEFAULT_SECURITY_PROTOCOL,
+          in(Utils.enumOptions(classOf[SecurityProtocol]):_*),
```

Comment:
This is very interesting. I got thrown off by reading this after seeing `in(Utils.enumOptions(SecurityProtocol.class))` because I couldn't understand why do we unpack the arguments in one of the situations, but not in the other. I suspect this is because in the Java files we don't use the Scala `in` function?

Reply:
Actually referring to the `ConfigDef.ValidString.in(String... validStrings)` method.

Reply:
Ah, I see. The Scala file imports the `in` method https://github.com/apache/kafka/blob/d47c7590ecd377f31b2d20acbcf065f502554252/core/src/main/scala/kafka/admin/BrokerApiVersionsCommand.scala#L32. Then I am confused - why do we need to unpack the arguments to this method in the Scala files, but not unpack them in the Java ones?

Reply:
Maybe Scala does not support indefinite parameters, that is `String... param`

Reply:
Fascinating, I wasn't aware of this. Okay, this makes sense as well.